homeassistant/components/blebox/sensor.py

```diff
@@ -75,3 +95,11 @@ def __init__(
     def native_value(self):
         """Return the state."""
         return self._feature.native_value
+
+    @property
+    def last_reset(self):
+        """Return the time when the sensor was last reset, if implemented."""
+        native_implementation = getattr(self._feature, "last_reset", None)
+        if callable(native_implementation):
+            return native_implementation
```

Comment:
You probably want to call the function?

```suggestion
            return native_implementation()
```

Reply:
This was my bad implementation, as this is property.